Fixed typos

PaulMcClernan opened the pull request:
from @seaniepie dropbox list
not ok - param amount in syntax section read from process command 
not ok - param string in syntax section read from process command

PaulMcClernan committed:
Fixed typos

from @seaniepie dropbox list
not ok - param amount in syntax section read from process command 
not ok - param string in syntax section read from process command

seaniepie commented:
Hi @PaulMcClernan , I've just noticed a number of other typos and errors in this page perhaps you could fix for us. 
- [ ] In parameters, the _processName_ should be a string (so make it _ProcessName (string):_ )
- [ ] The last word in the Important note in _processName_ should read 'open process command' with a space inserted.
- [ ] In parameters, _start_ is a string; _chunkType_ is an enum; _amount_ is an int
- [ ] At the end of the _Important_ note at the beginning of the _Description_ there is `<open processcommand>` which is not displaying because it should really be `<open process> <command>`
- [ ] The sections under this, `The <start>...` down to `The <time>` is all a duplicate of what is displayed in the Parameters section so can be removed.
- [ ] The next paragraph which says `The until <string> form` should probably be something like `The **until** form`, because the item `<string>` is just one of the 4 enums to pick from for the _until form_.
- [ ] The `<openProcesses function>` again should read `<openProcesses> <function>`
- [ ] You may choose to remove some of the references that were used in the chunk of description you delete as they will just clutter up the references section with unnecessary links.

See how you get on. Sorry it's so much. It seems you picked a doozy to work on.

PaulMcClernan committed:
Update read-from-process.lcdoc

PaulMcClernan commented:
@seaniepie 
The important note was actually a duplicate too. I chose to keep the descriptions with the glossary link / tags.

livecodeali commented on docs/dictionary/command/read-from-process.lcdoc:
```diff
@@ -26,41 +26,39 @@ Example:
-int4, or uint4. The read from process command reads amount of the
-specified chunkType. If you don't specify a chunkType, amount characters
-are read.
+chunkType (enum):
```
You should remove `(enum)` from this for now, as I don't think this entry is best formatted in the enum style.

livecodeali commented on docs/dictionary/command/read-from-process.lcdoc:
```diff
@@ -83,37 +81,13 @@ Description:
+The until end and until eof forms read data until the
+process specifies it is finished. 
+When until empty is used the data is read in at specified time intervals.
+If, when checked, the data is empty, the <openProcesses function> is used 
```
This needs to be `<openProcesses> <function>`

livecodeali commented:
Ah I see @seaniepie has been advising on this one 😄 

`openProcesses function (function)` and `open processcommand (command)` need to be removed from the references.

@seaniepie @PaulMcClernan It's important that `enum` parameters are formatted correctly if they are marked as enum. (see here https://github.com/livecode/livecode/blob/develop/docs/guides/LiveCode%20Documentation%20Format%20Reference.md#parameters)

PaulMcClernan committed:
Update read-from-process.lcdoc

removed (enum)
<openProcesses> <function>
openProcesses function (function) and open processcommand (command) need to be removed

livecodeali commented:
@livecode-vulcan review ok 7087929

livecode-vulcan commented:
💙 review by @livecodeali ok 70879295bc3e37b360e308a07ab71fce59b9ce29

livecode-vulcan commented:
😎 test success 70879295bc3e37b360e308a07ab71fce59b9ce29
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/466)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/471)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/466)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/466)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/472)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/466)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/466)
